### PR TITLE
Story dev storybook deployment to a publicly available address 181866678

### PR DESCRIPTION
**Pull request by iz-podpolja**
* Semaphore config for the deployment to npm & github pages
* Small build adjustments
* License file that was missing

### CODE REVIEW

File: .semaphore/npm.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+version: v1.0
+name: Deployment
+agent:
+  machine:
+    type: e1-standard-2
+    os_image: ubuntu1804
```
Comment (flori) [inline .semaphore/npm.yml:6]: Can you upgrade these to `ubuntu2004`?

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+To apply the Apache License to your work, attach the following
+boilerplate notice, with the fields enclosed by brackets "[]"
+replaced with your own identifying information. (Don't include
+the brackets!)  The text should be enclosed in the appropriate
+comment syntax for the file format. We also recommend that a
+file or class name and description of purpose be included on the
+same "printed page" as the copyright notice for easier
+identification within third-party archives.
+
+Copyright [yyyy] [name of copyright owner]
```
Comment (flori) [inline LICENSE:189]: I think you're supposed to enter betterplace and the current year here.

**Review by flori — APPROVED** [review]
I think this should work, but I didn't test publishing to NPM obviously. 😃

File: LICENSE (hunk 2 above)
Comment (iz-podpolja) [reply LICENSE:189]: Yep, haven't noticed this. Thanks

**Pull request merged**